### PR TITLE
Add support for expanded widget properties

### LINKED ISSUE #124: Update code to match new widgets API functionality
The API now returns widget objects with `order` and `aggregation` fields.
1. Milia needs to support setting these upon creation.
2. Milia also needs to support updating properties via PATCH.

### PR DESCRIPTION
**Pull request by okal**
Closes #124.

### CODE REVIEW

File: src/milia/api/widgets.cljc [hunk 1]
```diff
@@ -28,7 +27,10 @@
     `:widget_type` is a string, determined by the client e.g. chart
     `:view_type` is a string, determined by the client e.g. bar-chart
     `:column` is the  data column to be stored based on the form field.
-    `:group_by` the data column for the data to be grouped by. Optional"
+    `:order` this is the position of the widget in relation to others
```
Comment (pld) [inline src/milia/api/widgets.cljc]: is this optional too?

Comment (okal) [inline src/milia/api/widgets.cljc]: order? Yeah, it should be automatically added on the API side. We only ever need to set it manually when re-ordering.

Comment (pld) [inline src/milia/api/widgets.cljc]: Cool, yeah just checked the tests to make sure, lets also add "Optional" at the end of this arg description

**Comment by pld** [comment]
:+1:

**Pull request merged**